### PR TITLE
- common styles

### PR DESCRIPTION
**Pull request by irenelagno**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
Before: https://main--hsf-commonmoves--hlxsites.hlx.page/
After: https://blog-common-styles--hsf-commonmoves--hlxsites.hlx.page/

### CODE REVIEW

**Comment by irenelagno** [comment]
@bstopp can you have a look?

File: styles/styles.css [hunk 1]
```diff
@@ -18,14 +18,25 @@
...
+  --grey: #d8d8d8;
   --white: #fff;
+  --greyblue: #36758d;
   --white-background: #fff;
-  --border-light-gray: #d8d8d8;
-  --border-medium-gray: #aaa;
+  --border-light-grey: #d8d8d8;
+  --border-medium-grey: #aaa;
   --primary-color-svg-filter: invert(14%) sepia(47%) saturate(1309%) hue-rotate(272deg) brightness(96%) contrast(89%);
+  --seller-advice: rgb(240 240 224);
```
Comment (bstopp) [inline styles/styles.css:31]: Are these used anywhere besides blog? Can they be referenced from a blog css and imported into the appropriate sub-blocks?

File: styles/styles.css [hunk 2]
```diff
@@ -119,14 +130,22 @@ h4 {
...
+p, span {
   font-family: var(--font-family-proxima);
   font-size: var(--body-font-size-m);
   font-weight: var(--font-weight-normal);
   letter-spacing: var(--letter-spacing-reg);
   line-height: var(--line-height-m);
 }
 
+b, strong {
+  font-weight: bolder;
```
Comment (bstopp) [inline styles/styles.css:142]:
```suggestion
  font-weight: var(--font-weight-bold);
```

**Review by bstopp — CHANGES_REQUESTED** [review]

**Review by bstopp — APPROVED** [review]

File: styles/styles.css (hunk 1 above)
Comment (irenelagno) [reply styles/styles.css:31]: I'm still determining. maybe we can do the second round of CSS clean up at the end of the project?

Comment (bstopp) [reply styles/styles.css:31]: Sounds good.

**Review by bstopp — APPROVED** [review]

**Pull request merged**